e2e/ansible/roles/k8s-openebs-operator/tasks/main.yml

```diff
@@ -9,6 +9,12 @@
   delay: 5
   retries: 3
 
+- name: Change the ImagePullPolicy of openebs resources to Always in operator YAML
```

Comment:
Make the same change for the `openebs_cast_alias` as well.

Reply:
Also, can `openebs_cast_alias` be renamed to `openebs_pre_release_alias` to keep it generic for upcoming releases as well.

Reply:
The ImagePullPolicy Change for the CAS Templates?